src/code/z_sram.c

```diff
@@ -448,9 +448,13 @@ void Sram_OpenSave(SramContext* sramCtx) {
         }
     }
 
-    if (LINK_AGE_IN_YEARS == YEARS_ADULT && !CHECK_OWNED_EQUIP(EQUIP_TYPE_SWORD, EQUIP_INV_SWORD_MASTER)) {
+    //! Restore Master Sword to inventory and re-equip it if missing. Patch for one way of getting the "swordless Link"
+    //! glitch
```

Comment:
we can probably put the note about the patch on its own line, and phrase it as "This is a patch for one way...". also why the exclamation marks? we really only do that for bugs